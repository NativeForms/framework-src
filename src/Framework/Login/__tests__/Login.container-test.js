import React from 'react';
import { shallow } from 'enzyme';
import LoginContainer from '../Login.container';

describe('container <LoginContainer />', () => {
  it('container render correctly', () => {
    const wrapper = shallow(
      <LoginContainer />
    );
    expect(wrapper.dive()).toMatchSnapshot();
  });
});

