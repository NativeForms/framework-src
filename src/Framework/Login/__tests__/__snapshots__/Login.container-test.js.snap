// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container <LoginContainer /> container render correctly 1`] = `
<Content
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
    }
  }
>
  <FormComponent
    messages={
      Object {
        "login": "Sign In",
        "moreInfo": "Click for more info",
        "password": "Password",
        "title": "Login",
        "username": "Username",
      }
    }
    schema={
      Object {
        "attributes": Object {
          "formComponents": Array [
            Object {
              "attributes": Object {
                "headerLabel": Object {
                  "code": "username",
                },
              },
              "category": "field",
              "type": "text",
              "uid": "username",
            },
            Object {
              "attributes": Object {
                "headerLabel": Object {
                  "code": "password",
                },
                "hiddenText": true,
              },
              "category": "field",
              "type": "text",
              "uid": "password",
            },
            Object {
              "attributes": Object {
                "label": Object {
                  "code": "login",
                },
              },
              "category": "control",
              "type": "button",
              "uid": "login",
            },
            Object {
              "attributes": Object {
                "label": Object {
                  "code": "moreInfo",
                },
                "route": "https://nativebase.io/",
              },
              "category": "control",
              "type": "link",
              "uid": "moreInfo",
            },
          ],
          "label": "",
        },
        "category": "group",
        "type": "section",
        "uid": "loginExample",
      }
    }
  />
</Content>
`;
